cvmfs/cache_ram.h

```diff
@@ -0,0 +1,191 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+#ifndef CVMFS_CACHE_RAM_H_
+#define CVMFS_CACHE_RAM_H_
+
+#include <pthread.h>
+
+#include <cassert>
+#include <cstdlib>
+#include <vector>
+
+#include "statistics.h"
+#include "cache.h"
+#include "hash.h"
+#include "util/pointer.h"
+#include "kvstore.h"
+
+namespace cache {
+
+/**
+ * ...
```

Comment:
Here it would be good to have a short description of the purpose of the RamCacheManager and a brief sketch how it is implemented.